packages/jest-runner/src/testWorker.ts

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import {parseError, printError, printNode} from '@stack-tools/node-tools';
```

Comment:
thoughts on exporting some helpers from `jest-message-util` for this? A single `separateMessageFromStack` is easier than using these 3 functions, and keeping that complexity inside `jest-message-util` (like today) seems advantageous to me